test_scripts/Defects/5_0/1904_Media_app_activation_during_active_embedded_audio_source.lua

```diff
@@ -0,0 +1,79 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/1904
+--
+-- Description:
+-- Media app activation during active embedded audio source
+-- Precondition:
+-- SDL and HMI are started.
```

Comment:
Remove item `SDL and HMI are started`.
Remove `Precondition` section and move others actual steps to section `In case`